1-the-basics/style.css

```diff
@@ -0,0 +1,22 @@
+header, footer{
+    background-color: coral;
+}
+form{
+    background-image: url(https://dissenyaweb.com/blog/diseno_web/wp-content/uploads/large-background-1024x724.jpg);
+    background-size: cover;
+    background-color: darkgoldenrod;
+}
+button:hover{
+    background-color: darkorange;
+}
+aside{
+    font-size: larger;
+}
+p{
+    font-family: 'Segoe UI', Arial, Helvetica, sans-serif;
+    color: white;
+}
+
+.font{
```

Comment:
This class is not being applied to any element in your html, use it or remove it.

Reply:
remove this class, is not being used.